### PR TITLE
feature: bot dotenv

### PR DESCRIPTION
**Pull request by leon3s**
It's a pain on windows to add env variable so i added a dotenv.
Another solution would have been to use `clap` maybe and take command line arguments!
I also changed the code a bit so it's executed in parallel ! It's an equivalent of a `Promise.all` in javascript!

### CODE REVIEW

File: bot/src/main.rs [hunk 1]
```diff
@@ -13,21 +14,19 @@ use chrono::Utc;
 
 #[tokio::main]
 async fn main() {
+    dotenv::dotenv().ok();
```
Comment (darioalessandro) [inline bot/src/main.rs:17]: makes sense! is there a way to signal this to users in the README?

File: bot/src/main.rs [hunk 2]
```diff
@@ -13,21 +14,19 @@ use chrono::Utc;
...
-    let mut clients = Vec::new();
-    for _ in 0..n_clients {
-        clients.push(create_client(&endpoint, &room, &echo_user, &email_prefix).await);
-    }
-
-    for client in clients {
-        client.await;
-    }
+    (0..n_clients)
+    .into_iter().map(|_| async {
```
Comment (darioalessandro) [inline bot/src/main.rs:26]: neat!!!

**Review by darioalessandro — APPROVED** [review]
lgtm! imo it is worth throwing a line in the README about this: https://github.com/security-union/zoom-rs/tree/main/bot

**Comment by leon3s** [comment]
I guess i'm allowed to merge when approved ?

**Comment by darioalessandro** [comment]
> I guess i'm allowed to merge when approved ?

Yes, but you should accept the invite to become a collaborator, you should have it in your email inbox.

**Comment by leon3s** [comment]
Yes i accepted it!

**Comment by darioalessandro** [comment]
@leon3s thank you for all the awesome features!!! you rock!!

**Pull request merged**